doc/tutorial.md

```diff
@@ -125,62 +125,60 @@ st.msg("This is a message from dds to ros2");
 ```
 
 #### `HelloWorldSubscriber.cxx`
-Analogous to the first publisher modification, search the `topicName` line
+Analogously to the above publisher modification, search the `topicName` line of the subscriber
 ```c++
-Wparam.topic.topicName = "Vector2iSub";
+Wparam.topic.topicName = "HelloWorldPubSubTopic";
 ```
-and modify with the following topic Name:
+and modify it with a different topic name, for instance the following:
 ```c++
 Wparam.topic.topicName = "hello_dds";
 ```
 
 Once the files have been modified, we need to compile them.
 Simply, write `make` in your build folder.
 
-If we run now the publisher and subscriber, they will not match among them.
+If we now run the dds world publisher and subscriber, they will not match between them.
 
 
 ### Deploy
-For deploy all the system we will need 5 terminals at the same time:
+To deploy the system we will need 5 terminals at the same time:
 ![](images/terminal_deploy.png)
 
 #### Terminal 1
-To publish the topic from ros2, only need to write the following commands:
+To publish the topic from ros2, we only need to write the following commands:
 ```
-source ~/ros2_eloquent/ros2-linux/setup.bash
-ros2 topic pub /hello_dds std_msgs/String "This is a message from ros2 to dds"
+source ~/ros2_eloquent/install/setup.bash
+ros2 topic pub /hello_dds std_msgs/String "{\"This is a message from ros2 to dds\"}"
 ```
 
 #### Terminal 2
-Similar to publish from ros2, for subscribe we need to write:
+Similarly to what we did to publish from ros2, for subscribing we need to write:
 ```
-source ~/ros2_eloquent/ros2-linux/setup.bash
+source ~/ros2_eloquent/install/setup.bash
 ros2 topic echo /hello_ros std_msgs/String
 ```
 
 #### Terminal 3
-Placed into the build folder when you have generated the HelloWorld publisher and subscriber.
-Run the publisher
+Placed into the build folder where you have generated the HelloWorld publisher and subscriber, run the dds world publisher
 ```
 ./HelloWorld publisher
 ```
 
 #### Terminal 4
-Similar to terminal 4, but running the subscriber:
+As in terminal 3, run the dds world subscriber:
 ```
 ./HelloWorld subscriber
 ```
 
 #### Terminal 5
-Before running soss, any matched is made.
-Placed into the `soss` workspace.
-Source both, ros2 and your installation, and run the `soss` executable with the previous yaml.
+All matchings are made before running soss.
```

Comment:
This sentence means the opposite, maybe I have wrongly expressed my self. I want to say that no matching has be done before soss

Reply:
Ah ok. So maybe we could change it to: "No matching is made before running soss.".

Reply:
Ok perfect!